sdk/python/feast/feature_set.py

```diff
@@ -306,6 +306,33 @@ def _add_fields(self, fields: List[Field]):
         for field in fields:
             self.add(field)
 
+    def _entities_and_features_to_fields(
```

Comment:
Why is this a method and not a function?

Reply:
It doesn't seem like this will be reused.

Reply:
Ok, at first putting underscore is a private function. Because it is only being used inside this class, have just re-read about method and function, I am rewriting it to a function.

Thanks